<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>containingWidget</class>
 <widget class="QWidget" name="containingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;b&gt;Problem specific labels:&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabelListView" name="labelListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="segment">
     <property name="text">
      <string>Segment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveAs">
     <property name="text">
      <string>Save As ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="load">
     <property name="text">
      <string>Load ...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>bg priority</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="backgroundPrioritySpin">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.005000000000000</double>
       </property>
       <property name="value">
        <double>0.950000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>no bias below</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="noBiasBelowSpin">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>64</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="arrowToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="paintToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="eraserToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="brushSizeComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddLabelButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="brushSizeCaption">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelListView</class>
   <extends>QTableView</extends>
   <header>igms.labelListView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
